docs/upgrade.html

```diff
@@ -48,6 +48,11 @@ <h5><a id="upgrade_400_notable" href="#upgrade_400_notable">Notable changes in 4
                 The <code>--whitelist</code> option was removed from the <code>kafka-console-consumer</code> command line tool.
                 Please use <code>--include</code> instead.
             </li>
+            <li>
+                The <code>blacklist/whitelist</code> was removed from the <code>kafka.common.metrics.JmxReporter</code>
```

Comment:
please use full name instead of blacklist/whitelist/EXCLUDE_CONFIG/INCLUDE_CONFIG